src/storage/hummock_sdk/src/key.rs

```diff
@@ -654,8 +654,8 @@ impl<T: AsRef<[u8]>> Debug for FullKey<T> {
             f,
             "FullKey {{ {:?}, epoch: {}, epoch_with_gap: {}}}",
             self.user_key,
+            self.epoch_with_gap.pure_epoch(),
```

Comment:
In addition to the u64 pure epoch and the u64 epoch with gap, can we log the spill offset for debugging convenience as well?

Reply:
+1